cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1146,6 +1346,18 @@ function Configure-Kubelet {
   Log-Output "Kubelet config:`n$(Get-Content -Raw ${env:KUBELET_CONFIG})"
 }
 
+function Configure-Antrea {
+  $antrea_config = 'trafficEncapMode: noEncap
+featureGates:
+  AntreaPolicy: false
+  Traceflow: false
+clientConnection:
+  kubeconfig: \etc\kubernetes\antrea.kubeconfig
```

Comment:
prefix with `C:` or use `/` delimiters.

Reply:
@jeremyje  `\etc\kubernetes\antrea.kubeconfig` works. 
This is the way the Antrea team uses it.